ui/src/styles.css

```diff
@@ -90,3 +89,154 @@ input[type=text]:focus, input[type=password]:focus {
 .noUi-connect {
   background: rgb(92, 183, 225);
 }
+
+/*
+ * Override clarity fonts
+ */
+body {
```

Comment:
so FYI, if the rule is the same for all these selectors, they can just be combined with commas.  I.e.

```
.login-wrapper .login .title,
.login-wrapper .login .trademark,
.modal-header .modal-title,
/* etc */
h1, h2, h3, h4, h5, h6 {
  /* font-family rule or font-weight rule or what have you */
}
```

That might help clean this up a little bit.